src/components/Cluster/KubernetesTableShow/index.js

```diff
@@ -321,11 +322,9 @@ export default class KubernetesClusterShow extends PureComponent {
                   <a>删除</a>
                 </Popconfirm>
               )}
-            {row.state === 'running' && selectProvider === 'rke' && (
-              <a onClick={() => this.updateCluster(row.cluster_id || row.name)}>
-                节点扩容
-              </a>
-            )}
+            <a onClick={() => this.updateCluster(row.cluster_id || row.name)}>
```

Comment:
should not remove `selectProvider === 'rke'`.

Reply:
Have to repair